src/cairoWriter/utils.ts

```diff
@@ -24,10 +24,12 @@ export function writeImports(imports: Map<string, Set<string>>): string {
     );
   }
   return [...imports.entries()]
-    .map(
-      ([location, importedSymbols]) =>
-        `from ${location} import ${[...importedSymbols.keys()].join(', ')}`,
-    )
+    .map(([location, importedSymbols]) => {
+      const toImport = [...importedSymbols.keys()];
+      const toImportFormatted =
+        toImport.length === 1 ? toImport[0] : '{' + toImport.join(', ') + '}';
```

Comment:
As we discussed rust style multi imports don't work in cairo 1. AFAIK @cicr99 is going to remove grouped imports so the PR have to be merged with Carmen's changes first

Reply:
That's a good idea

Reply:
As far as I see in @cicr99's PR she is taking care of this writters so i'm just gonna revert the changes so she face with no merge conflict with this ones